src/client/base/tokio.rs

```diff
@@ -102,7 +102,40 @@ impl TokioClient {
         Box::pin(async move {
             let bytes = send_inner(&client, request, &strategy).await?;
             let json_deserializer = &mut serde_json::Deserializer::from_slice(&bytes);
-            serde_path_to_error::deserialize(json_deserializer).map_err(StripeError::from)
+
+            match serde_path_to_error::deserialize(json_deserializer) {
```

Comment:
I would like to, instead, integrate `tracing` into this crate properly. If you have the time to set this up, some TRACE level statements in the event of a failure would probably be well received.